sopel/modules/uptime.py

```diff
@@ -18,6 +18,13 @@ def setup(bot):
         bot.memory["start_time"] = datetime.datetime.utcnow()
 
 
+def shutdown(bot):
+    try:
+        del bot.memory['start_time']
+    except KeyError:
+        pass
```

Comment:
Don't clear this one. Doing so will make it so reloading the `uptime` module resets the bot's uptime to 0.

Reply:
Ah yes, good call.